<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="62" started-at="2014-11-19T18:23:14Z" finished-at="2014-11-19T18:23:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="62" started-at="2014-11-19T18:23:14Z" finished-at="2014-11-19T18:23:14Z">
      <class name="Tests.WhenLoginPanelIsDisplayed">
        <test-method status="FAIL" signature="ConnectionWithApplication()[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ConnectionWithApplication" is-config="true" duration-ms="0" started-at="2014-11-19T19:22:19Z" finished-at="2014-11-19T19:22:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Problem with create Webdrive instantion: org.openqa.selenium.UnsupportedCommandException: ERROR The test with session id 205b8fd52a5e469eb04adcc17900b595 has already finished, and can't receive further commands
You can learn more at https://saucelabs.com/jobs/205b8fd52a5e469eb04adcc17900b595
For help, please check https://docs.saucelabs.com/reference/troubleshooting-common-error-messages/
Command duration or timeout: 54.36 seconds
Build info: version: '2.43.1', revision: '5163bce', time: '2014-09-10 16:27:33'
System info: host: 'rafal', ip: '192.168.1.4', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_71'
Driver info: org.openqa.selenium.remote.RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Problem with create Webdrive instantion: org.openqa.selenium.UnsupportedCommandException: ERROR The test with session id 205b8fd52a5e469eb04adcc17900b595 has already finished, and can't receive further commands
You can learn more at https://saucelabs.com/jobs/205b8fd52a5e469eb04adcc17900b595
For help, please check https://docs.saucelabs.com/reference/troubleshooting-common-error-messages/
Command duration or timeout: 54.36 seconds
Build info: version: '2.43.1', revision: '5163bce', time: '2014-09-10 16:27:33'
System info: host: 'rafal', ip: '192.168.1.4', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_71'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
	at org.testng.Assert.fail(Assert.java:94)
	at Base.ShareBase.ConnectionWithApplication(ShareBase.java:48)
	at Base.ShareBase.ConnectionWithApplication(ShareBase.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:277)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ConnectionWithApplication -->
        <test-method status="SKIP" signature="setUp()[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="setUp" is-config="true" duration-ms="0" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="ShoulDisplayLoginPanel()[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ShoulDisplayLoginPanel" duration-ms="1" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Base.ShareBase.getSessionId(ShareBase.java:117)
	at com.saucelabs.testng.SauceOnDemandTestListener.onTestStart(SauceOnDemandTestListener.java:93)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1904)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:685)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ShoulDisplayLoginPanel -->
        <test-method status="SKIP" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ScreenshotWhenFail" is-config="true" duration-ms="0" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="FAIL" signature="ShouldDisplayAllCOntrols()[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ShouldDisplayAllCOntrols" duration-ms="1" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Base.ShareBase.getSessionId(ShareBase.java:117)
	at com.saucelabs.testng.SauceOnDemandTestListener.onTestStart(SauceOnDemandTestListener.java:93)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1904)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:685)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ShouldDisplayAllCOntrols -->
        <test-method status="SKIP" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ScreenshotWhenFail" is-config="true" duration-ms="0" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="FAIL" signature="ShouldDisplayMessageWhenUserSendNotReginstedEmail()[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ShouldDisplayMessageWhenUserSendNotReginstedEmail" duration-ms="0" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Base.ShareBase.getSessionId(ShareBase.java:117)
	at com.saucelabs.testng.SauceOnDemandTestListener.onTestStart(SauceOnDemandTestListener.java:93)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1904)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:685)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ShouldDisplayMessageWhenUserSendNotReginstedEmail -->
        <test-method status="SKIP" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ScreenshotWhenFail" is-config="true" duration-ms="0" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="FAIL" signature="ShouldLogWhenUserEnterCorrectData()[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ShouldLogWhenUserEnterCorrectData" duration-ms="0" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at Base.ShareBase.getSessionId(ShareBase.java:117)
	at com.saucelabs.testng.SauceOnDemandTestListener.onTestStart(SauceOnDemandTestListener.java:93)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1904)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:685)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ShouldLogWhenUserEnterCorrectData -->
        <test-method status="SKIP" signature="ScreenshotWhenFail(org.testng.ITestResult)[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="ScreenshotWhenFail" is-config="true" duration-ms="0" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScreenshotWhenFail -->
        <test-method status="SKIP" signature="CloseConnection()[pri:0, instance:Tests.WhenLoginPanelIsDisplayed@8ba6621]" name="CloseConnection" is-config="true" duration-ms="0" started-at="2014-11-19T19:23:14Z" finished-at="2014-11-19T19:23:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseConnection -->
      </class> <!-- Tests.WhenLoginPanelIsDisplayed -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
